perfkitbenchmarker/pkb.py

```diff
@@ -506,12 +503,18 @@ def RunBenchmarks(publish=True):
   for i, benchmark_tuple in enumerate(benchmark_tuple_list):
     benchmark_module, user_config = benchmark_tuple
     benchmark_name = benchmark_module.BENCHMARK_NAME
+    benchmark_config_dict = benchmark_module.GetConfig(user_config)
```

Comment:
Consider making a function that turns a dictionary into a benchmark config spec, either here or (ideally) in benchmark_config_spec.py.